tests/test_files/Model/Entity/Constants/Wheel.php

```diff
@@ -11,7 +11,7 @@
  * @property \Cake\I18n\FrozenTime|null $modified
  * @property \TestApp\Model\Entity\Wheel[] $wheels
  *
- * @property-read string|null $virtual_one
+ * @property-read string|null $virtual_one {@uses \TestApp\Model\Entity\Wheel::_getVirtualOne()}
```

Comment:
What about set_* method instead of just the getter?
Less likely, but still possible, right?

If we focused on the annotation of the methods themselves I think we could solve both together.